doc/languages-frameworks/python.section.md

```diff
@@ -2021,9 +2021,22 @@ The following rules are desired to be respected:
   and using a `-` as delimiter.
 * Attribute names in `python-packages.nix` should be sorted alphanumerically to
   avoid merge conflicts and ease locating attributes.
+* The list of [dependencies to avoid or replace](#python-dependencies-avoid-replace) is respected.
 
 This list is useful for reviewers as well as for self-checking when submitting packages.
 
+## Dependencies to avoid or replace {#python-dependencies-avoid-replace}
```

Comment:
To scope it within nixpkgs contributions, rather than for example downstream usage like ad-hoc shells
```suggestion
### Dependencies to avoid or replace {#python-dependencies-avoid-replace}
```